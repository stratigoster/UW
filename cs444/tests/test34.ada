PACKAGE foo IS
  function "+" (a,b:integer) return boolean;
  function "+" (a,b:integer) return integer; -- overloaded '+'

  function "+" (c,d:integer) return boolean; -- duplicate
-- this is alpha-equivalent to the function declaration in line 2
END foo;
