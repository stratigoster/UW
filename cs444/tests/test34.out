***ERROR***:5:45:	Conflicting declaration `+:function (integer,integer) return universal boolean'

------
Compilation error
------
